doc/kconfig/conf.py

```diff
@@ -59,7 +58,14 @@
 # If true, license is shown in the HTML footer. Default is True.
 html_show_license = True
 
+html_theme_options = {
+    "versions": {
+        "ncs": "nRF Connect SDK",
+        "nrfxlib": "nrfxlib",
+        "zephyr": "Zephyr",
+        "mcuboot": "MCUboot"
+    }
+}
+
 def setup(app):
-    app.add_css_file("css/common.css")
     app.add_css_file("css/kconfig.css")
-    app.add_js_file("js/ncs.js")
```

Comment:
ncs.js still must be included in some way (in all doc sets). Currently it isn't, which means all our JavaScript modifications are missing.

Reply:
fixed on 0.4.0 release

Reply:
No idea why, but it seems to be included twice now.

Reply:
This may be a Sphinx bug, as only a single call to `add_js_file` is made. Can we investigate this later? Does not seem to affect right now.

Reply:
Sure.